crates/edr_evm/src/chain_spec.rs

```diff
@@ -26,40 +26,34 @@ use crate::{
 #[allow(clippy::trait_duplication_in_bounds)]
 pub trait ChainSpec:
     alloy_rlp::Encodable
+    // Defines the chain's internal types like blocks/headers or transactions
     + EthHeaderConstants
     + EvmWiring<
         Block: BlockEnvConstructor<Self, block::Header> + BlockEnvConstructor<Self, PartialHeader>,
         Transaction: alloy_rlp::Encodable
-                         + Clone
-                         + Debug
-                         + PartialEq
-                         + Eq
-                         + ExecutableTransaction
-                         + Transaction
-                         + TransactionType
-                         + TryFrom<
-            <Self as RpcSpec>::RpcTransaction,
-            Error = Self::RpcTransactionConversionError,
-        >,
-    > + RpcSpec<
-        ExecutionReceipt<FilterLog>: Debug,
+          + Clone
```

Comment:
I like the alignment better. I'm curious how you achieved it?

I'm using `cargo +nightly fmt` but it formatted it the way it was before.

Reply:
I think we should pin the nightly version used for formatting for consistency. I noticed that formatting wasn't working for me when I tried formatting with `cargo +nightly fmt` (was using `rustc 1.82.0-nightly (e57f3090a 2024-08-05)` directly or with the git pre-commit hook script but nothing worked, so I formatted it manually. I now bumped the nightly version to `rustc 1.83.0-nightly (d6c8169c1 2024-09-03)` and still can't force rustfmt to reformat this.

What exactly is your process to format these?

Reply:
I'm using VSCode's "format on save" and have it configured to use the nightly formatter.